src/components/Personalization/index.js

```diff
@@ -93,6 +141,20 @@ const createPersonalization = ({ config, logger, eventManager }) => {
 
         collectClicks(merger, clickedElement, storage);
       }
+    },
+
+    commands: {
+      getDecisions(options = {}) {
+        validateOptions(options);
+
+        const { viewStart, scopes = [] } = options;
```

Comment:
if you want to apply the defaults, you need to use the object returned from the validator.

Reply:
yes, thank you